src/extensions/timeseries/TimeseriesViewNew/TimeseriesViewNew.tsx

```diff
@@ -64,13 +66,14 @@ const TimeseriesViewNew = (props: Props) => {
                                 recordingInfo={props.recordingInfo}
                                 width={0} // filled in above
                                 height={0} // filled in above
+                                visibleElectrodeIds={visibleElectrodeIds}
                                 selectedElectrodeIds={selectedElectrodeIds}
                                 onSelectedElectrodeIdsChanged={handleSelectedElectrodeIdsChanged}
                             />
                         )
                     }
                     {
-                        ((!opts.channelSelectPanel) || (selectedElectrodeIds.length > 0)) ? (
+                        ((!opts.channelSelectPanel) || (selectedElectrodeIds.length > 0) || (visibleElectrodeIds.length <= 12)) ? (
```

Comment:
If no electrodes are selected: show them all if fewer than 12, otherwise show none and prompt user to select electrodes